github4s/shared/src/main/scala/github4s/api/PullRequests.scala

```diff
@@ -80,8 +81,56 @@ class PullRequests[C, M[_]](
       owner: String,
       repo: String,
       number: Int): M[GHResponse[List[PullRequestFile]]] =
-    httpClient.get[List[PullRequestFile]](
-      accessToken,
-      s"repos/$owner/$repo/pulls/$number/files",
-      headers)
+    httpClient
+      .get[List[PullRequestFile]](accessToken, s"repos/$owner/$repo/pulls/$number/files", headers)
+
+  /**
+   * Create a pull request
+   *
+   * @param accessToken to identify the authenticated user
+   * @param headers optional user headers to include in the request
+   * @param owner of the repo
+   * @param repo name of the repo
+   * @param newPullRequest	Required. The head and body parameters or the issue parameter
+   * @param head		Required. The name of the branch where your changes are implemented. For cross-repository pull
+   *                             requests in the same network, namespace head with a user like this: username:branch.
+   * @param base		Required. The name of the branch you want the changes pulled into. This should be an existing branch
+   *                             on the current repository. You cannot submit a pull request to one repository that
+   * @param maintainerCanModify		Indicates whether maintainers can modify the pull request.
```

Comment:
Bad indentation, please remove the tabs in the scaladocs. Also, please specify that the value for `maintainerCanModify` is `true` by default